trunk/NDHMS/Data_Rec/module_namelist.F

```diff
@@ -62,6 +62,8 @@ subroutine read_rt_nlst(nlst)
           integer            :: reservoir_observation_lookback_hours
```

Comment:
FWIW, this module isn't used anymore (replaced by the Orchestrator). No harm updating it, but not strictly necessary.